### PR TITLE
US225619 pfe nav style updates

### PR DESCRIPTION
**Pull request by chrisdo1**
- removed padding beneath icons in utility nav
- fixed spacing between menu items

## PFE Nav style updates

* Link to discovery documentation (should contain all the planning and requirements for this work):
   - See the `DISCOVERY.md` template at the root of the project
* Link(s) to demo pages where this element can be viewed:

---

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

*

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. 

#### Browser requirements
Your component should work in all of the following environments:

- [ ] Latest 2 versions of Edge
- [ ] Internet Explorer 11 (should be useable, not pixel perfect)
- [ ] Latest 2 versions of Firefox (one on Mac OS, one of Windows OS)
- [ ] Firefox 60.7.2 (or latest version for Red Hat Enterprise Linux distribution)
- [ ] Latest 2 versions of Chrome (one on Mac OS, one of Windows OS)
- [ ] Latest 2 versions of Safari
- [ ] Galaxy S9 Firefox
- [ ] iPhone X Safari
- [ ] iPad Pro Safari
- [ ] Pixel 3 Chrome

Your repository infrastructure updates should work for at least:
- [ ] Node v8.x
- [ ] NPM v7.x

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Did browser testing pass?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?
- [ ] Did you update the CHANGELOG.md file with a summary of this update?

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

### CODE REVIEW

File: elements/pfe-navigation/src/_shared-assets.scss [hunk 1]
```diff
@@ -31,11 +31,19 @@ $USE-FALLBACK: true; // turn on automatic fallbacks for broadcasted variables
 
 %trigger {
     font-family: pfe-var(font-family);
-    font-size:   pfe-local($cssvar: FontSize, $region: trigger, $fallback: #{$trigger--FontSize--fallback});
+    font-size:   pfe-local($cssvar: FontSize, $region: trigger, $fallback: #{$trigger--FontSize--fallback}) !important;
```
Comment (starryeyez024) [inline elements/pfe-navigation/src/_shared-assets.scss]: the font-size fallback is set to grab the theme font size "zeta" which is 14px, but we are using 16px which is "epsilon".

File: elements/pfe-navigation/demo/index.html [hunk 2]
```diff
@@ -31,9 +31,12 @@
     ])
 
   </script>
+  <!-- <link type="text/css" rel="stylesheet" href="https://www.redhat.com/files/webux/campaign/itwob/bootstrap.css" media="all" /> -->
```
[the remark this answers is not in the document]
Comment (chrisdo1) [reply elements/pfe-navigation/demo/index.html]: @starryeyez024 fixed.

File: elements/pfe-navigation/src/_shared-assets.scss [hunk 3]
```diff
@@ -31,11 +31,19 @@ $USE-FALLBACK: true; // turn on automatic fallbacks for broadcasted variables
...
     font-weight: pfe-local($cssvar: FontWeight, $fallback: #{$trigger--FontWeight--fallback});
     color:       pfe-local($cssvar: Color, $fallback: #{$trigger--Color--fallback});
+    background:  #{pfe-local($cssvar: Cover, $region: trigger, $fallback:  #{pfe-color(surface--darkest)})};
+    z-index: 2;
```
[the remark this answers is not in the document]
Comment (chrisdo1) [reply elements/pfe-navigation/src/_shared-assets.scss]: @starryeyez024 Good suggestion. Fixed.

File: elements/pfe-navigation/src/pfe-navigation-item.scss [hunk 4]
```diff
@@ -117,7 +137,7 @@ $variables: (
     }
   }
 
-  // After element is the filled-in dropdown caret
+  // After element is the filled-in dropdown caret.
```
[the remark this answers is not in the document]
Comment (chrisdo1) [reply elements/pfe-navigation/src/pfe-navigation-item.scss]: @starryeyez024 I think this looks better but I don't think the arrow will be covered when/if the trigger text extends to a second line.

[the remark this answers is not in the document]
Comment (chrisdo1) [reply elements/pfe-navigation/src/pfe-navigation-item.scss]: @starryeyez024 moved it down to 8px and tightened the line-height, so maybe we don't need to cover the caret if the trigger text wraps?

Screen Shot 2019-10-28 at 4 34 45 PM

File: elements/pfe-navigation/src/pfe-navigation-item.scss [hunk 5]
```diff
@@ -13,12 +13,20 @@ $variables: (
 
   --pfe-navigation__trigger-icon--Visible: hidden;
   --pfe-navigation__trigger--FontSize: 16px;
+  --pfe-navigation__trigger--Padding: calc(#{pfe-var(container-padding)} / 2) calc(#{pfe-var(container-padding)} *.75) !important;
```
[the remark this answers is not in the document]
Comment (chrisdo1) [reply elements/pfe-navigation/src/pfe-navigation-item.scss]: @starryeyez024 I tried this but the FontSize local variable isn't getting applied to the trigger. Is there any reason we can't move the %trigger placeholder into `pfe-navigation-item.scss`?

File: elements/pfe-navigation/src/_shared-assets.scss (hunk 1 above)
Comment (chrisdo1) [reply elements/pfe-navigation/src/_shared-assets.scss]: @starryeyez024 set this to use the header--epsilon font size.

File: elements/pfe-navigation/src/pfe-navigation-item.scss (hunk 5 above)
[the remark this answers is not in the document]
Comment (chrisdo1) [reply elements/pfe-navigation/src/pfe-navigation-item.scss]: I'm going to mark this resolved and address it in the next pr.

File: elements/pfe-navigation/src/pfe-navigation--lightdom.scss [hunk 6]
```diff
@@ -52,29 +105,106 @@
...
+        .pfe-navigation-item__trigger::after {
+            display: none !important;
+        }
+        [aria-expanded=true].pfe-navigation-item__trigger::after {
+            display: block !important;
+        }
+        &.expanded {
+            .pfe-navigation-item__trigger {
+                pfe-icon:not(.load-failed) svg.pfe-icon filter.pfe-icon feFlood.pfe-icon {
+                    flood-color: #151515 !important;
```
Comment (starryeyez024) [inline elements/pfe-navigation/src/pfe-navigation--lightdom.scss]:
```suggestion
                    flood-color: #{pfe-color(ui-complement)} !important;
```

File: elements/pfe-navigation/src/pfe-navigation--lightdom.scss [hunk 7]
```diff
@@ -52,29 +105,106 @@
...
+
+        &:hover {
+            --pfe-navigation__trigger-icon--Visible: visible;
+        }
+        pfe-icon {
+            background-color: transparent !important;
+            max-height: 20px;
+        }
+        pfe-icon:not(.load-failed) svg.pfe-icon filter.pfe-icon feFlood.pfe-icon {
+            flood-color: #fff !important;
```
Comment (starryeyez024) [inline elements/pfe-navigation/src/pfe-navigation--lightdom.scss]:
```suggestion
            flood-color: #{pfe-colo(darkest--text)} !important;
```

File: elements/pfe-navigation/src/pfe-navigation--lightdom.scss [hunk 8]
```diff
@@ -219,11 +352,11 @@ pfe-navigation-item [slot="trigger"] > a {
         }
 
         a {
-            color: #151515;
+            color: var(--pfe-theme--color--text, #151515);
```
Comment (starryeyez024) [inline elements/pfe-navigation/src/pfe-navigation--lightdom.scss]:
```suggestion
            color: pfe-color(text);
```

File: elements/pfe-navigation/src/pfe-navigation--lightdom.scss [hunk 9]
```diff
@@ -219,11 +352,11 @@ pfe-navigation-item [slot="trigger"] > a {
...
             text-decoration: underline;
 
             &:hover {
-                color: #151515;
+                color: var(--pfe-theme--color--text, #151515);
```
Comment (starryeyez024) [inline elements/pfe-navigation/src/pfe-navigation--lightdom.scss]:
```suggestion
                color: pfe-color(text);
```

**Pull request merged**